### PR TITLE
add oracle privilege loader

### LINKED ISSUE #9854: Load users and privileges from Oracle
Hi,

This issue is part of #8966  and to load the privileges from Oracle. You can refer to `MySQLPrivilegeLoader` to finish this one. BTW, please give a look at #9851 for a better implement.

### Task
- Implement `OraclePrivilegeLoader `
- Add a test class for it (#9852 will give some hints).

Have a good day. :)

### PR DESCRIPTION
**Pull request by zhujunxxxxx**
Fixes #9854.

Changes proposed in this pull request:
- add oracle privilege loader

### CODE REVIEW

**Comment by zhujunxxxxx** [comment]
@tristaZero this PR need you review. thanks.

File: shardingsphere-features/shardingsphere-authority/shardingsphere-authority-common/src/main/java/org/apache/shardingsphere/authority/loader/storage/impl/dialect/OraclePrivilegeLoader.java [hunk 1]
```diff
@@ -0,0 +1,189 @@
...
+                return PrivilegeType.CREATE_SEQUENCE;
+            case "CREATE TABLESPACE":
+                return PrivilegeType.CREATE_TABLESPACE;
+            case "CREATE USER":
+                return PrivilegeType.CREATE_USER;
+            case "CREATE VIEW":
+                return PrivilegeType.CREATE_VIEW;
+            case "SYSDBA":
+                return PrivilegeType.SUPER;
+            // TODO other privilege
```
Comment (tristaZero) [inline shardingsphere-features/shardingsphere-authority/shardingsphere-authority-common/src/main/java/org/apache/shardingsphere/authority/loader/storage/impl/dialect/OraclePrivilegeLoader.java:179]: So these todo privilege is expected in next PR?

**Review by tristaZero — CHANGES_REQUESTED** [review]

File: shardingsphere-features/shardingsphere-authority/shardingsphere-authority-common/src/main/java/org/apache/shardingsphere/authority/loader/storage/impl/dialect/OraclePrivilegeLoader.java (hunk 1 above)
Comment (zhujunxxxxx) [reply shardingsphere-features/shardingsphere-authority/shardingsphere-authority-common/src/main/java/org/apache/shardingsphere/authority/loader/storage/impl/dialect/OraclePrivilegeLoader.java:179]: yes，the privilege type of oracle is too many.

**Review by tristaZero — APPROVED** [review]
Nice.

**Pull request merged**